src/components/Header/index.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import Button from "@rescui/button";
+import { MobileMenu } from "./mobile-menu/";
+import { DesktopMenu } from "./desktop-menu";
+import KotlinLogo from "../../images/kotlin-logo.inline.svg";
+import * as style from "./header.module.css";
+
+const menuItems = [
+  {
+    url: "/structure/",
+    title: "Structure",
+  },
+  {
+    url: "/faq/",
+    title: "FAQ",
+  },
+];
+
+export type MenuItem = {
+  url: string;
+  title: string;
+};
+
+export const Header: React.FC = () => {
+  const [isMenuOpened, setIsMenuOpened] = React.useState<boolean>(false);
+
+  const toggleMenu = React.useCallback(() => {
+    setIsMenuOpened(!isMenuOpened);
+  }, [isMenuOpened]);
+
+  return (
+    <div className={style.header}>
+      <div className={style.nav}>
+        <div className={style.logo}>
+          <div className={style.logoImage}>
+            <KotlinLogo />
+          </div>
+          <div className={style.logoText}>
+            Kotlin <br className={style.mobileBr} /> Foundation
```

Comment:
Some of that should be linked to index page, i think. It isn't understand how to return back on desktop. Ask designer, pls.

Reply:
Logo is a link to main page now